src/options.html

```diff
@@ -0,0 +1,63 @@
+<html>
+  <head>
+    <link crossorigin="anonymous" href="https://assets-cdn.github.com/assets/github/index-55d89b4ebef0905d740bb0006ae466934d43cfa6f709e983b6a4b584236d848c.css" media="all" rel="stylesheet">
```

Comment:
It's certainly a good question. Probably wouldn't hurt to grab what we need/use from here and pull it local to the extension. As I said bellow, very sorry I haven't had a chance to look into these things. It's turned out to be an extremely busy end-of-the-year!